exercises/strings/replaceCharacter/replaceCharacter.js

```diff
@@ -1,26 +1,16 @@
-/**
- * Given an input string, a target character to find, and a replacement character,
- * returns a copy of the input string with the target character replaced by the
- * replacement character.
- *
- * @example
- * replaceCharacter('hello', 'l', '8'); // => 'he88o'
- *
- * @param {string} string - The string to replace a character in
- * @param {string} target - The target letter
- * @param {string} replaceWith - The replacement letter
- * @returns {string} A copy of the input string with target replaced by replaceWith
- */
+
+let string = 'Sofia Gonzalez Leal'
+let target = 'a'
+let replaceWith = '7'
 
 function replaceCharacter(string, target, replaceWith) {
-  // This is your job. :)
+  return newString = string.replace(target, replaceWith)
```

Comment:
1. You don't need to assign the return value of `string.replace` to a variable before returning it. You can just `return string.replace(...)`.
1. Try to do this _without_ using the built-in `string.replace`.